lusidtools/cocoon/validator.py

```diff
@@ -18,10 +18,16 @@ def check_allowed_value(self, allowed_values: list):
         """
         Checks that value exists in the provided list
 
-        :param list allowed_values: The list of allowed values
+        Parameters
+        ----------
+        allowed_values :    list
+                            The list of allowed values
 
-        :return: self
+        Returns
+        -------
+        self
```

Comment:
The type of self here is the Validator class.

Reply:
Now returns `Validator`